Header  header
int32   x
int32   y
int32   size
float32 prob_f
float32 prob_l
float32 prob_r
